packages/cli/src/environments/variables/variables.controller.ts

```diff
@@ -28,7 +29,7 @@ variablesController.use(EEVariablesController);
 variablesController.get(
 	'/',
 	ResponseHelper.send(async () => {
-		return VariablesService.getAll();
+		return Container.get(VariablesService).getAll();
```

Comment:
Inject as a dependency instead?

Reply:
for that we'd have to rewrite the controllers into service classes first